scripts/build-test-artifacts

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This script generates artifacts used to test the agent. It only has windows
+# functional tests, but will eventaully be extended to support others.
+
+mkdir -p ./out/test-artifacts
+
+GOOS=windows go test -tags functional -o ./out/test-artifacts/windows-simple-tests.exe -c ./agent/functional_tests/tests/generated/simpletests_windows/
```

Comment:
We should to build all the test artifacts or take in a GOOS to build for. When we are ready to, we can take advantage of all the test artifacts built.

Reply:
> take in a GOOS

The problem with this is that the output files are going to be different (.exe) and I'd rather be explicit instead of switching on the supplied operating system.

> build all the test artifacts

I went back and forth on whether or not to do this yet &mdash;   you're the second person to ask me about it so I'll go ahead and do that :smiley: